website/docs/r/image.html.markdown

```diff
@@ -58,11 +58,14 @@ The following arguments are supported:
     the image. Changing this forces a new resource to be created.
 * `location` - (Required) Specified the supported Azure location where the resource exists.
     Changing this forces a new resource to be created.
+* `zone_resilient` - (Optional) Is zone resiliency enabled?  Defaults to `false`.  Changing this forces a new resource to be created.
```

Comment:
we're trying to keep these sorted Required -> Optional and then Alphabetically within the fields; could we move this to below Tags (and above the `Note`?)